mobile/src/main/java/org/hawkular/client/android/backend/BackendPipes.java

```diff
@@ -60,19 +60,19 @@ private Paths() {
         public static final String ALERT_ACKNOWLEDGE = "alerts/ack";
         public static final String ALERT_NOTE = "alerts/note";
         public static final String ALERT_RESOLVE = "alerts/resolve";
-        public static final String ENVIRONMENTS = "inventory/deprecated/environments";
-        public static final String FEEDS = "inventory/deprecated/feeds";
-        public static final String FEED_METRICS = "inventory/deprecated/feeds/%s/resources/%s/metrics";
-        public static final String FEED_RESOURCES = "inventory/deprecated/feeds/%s/resources";
-        public static final String FEED_REC_RESOURCES = "inventory/deprecated/feeds/%s/resources/%s/children";
-        public static final String METRICS = "inventory/deprecated/%s/resources/%s/metrics";
+        public static final String ENVIRONMENTS = "inventory/traversal/type=e";
```

Comment:
feeds, feeds_resources, resources look good to me, I am not 100% sure about the others @metlos would be better for this, I haven't completelly wrapped my head around it yet